### PR TITLE
noScript button improvement, plus fix lint error

### PR DESCRIPTION
**Pull request by luixxiul**
- [x] Submitted a [ticket](https://github.com/brave/browser-laptop/issues) for my issue if one did not already exist.
- [x] Used Github [auto-closing keywords](https://help.github.com/articles/closing-issues-via-commit-messages/) in the commit message.
- [x] Ran `git rebase -i` to squash commits if needed.

addresses #1564

1. l10n
2. cursor: pointer

also indentation based on the style guide, fix lint error

### CODE REVIEW

File: app/extensions/brave/locales/en-US/app.properties [hunk 1]
```diff
@@ -43,6 +43,7 @@ reloadTab=Reload tab
 reloadButton.title=Reload page
 homeButton.title=Open Homepage
 stopButton.title=Stop loading
+noScriptButton.title=Allow scripts
```
Comment (luixxiul) [inline app/extensions/brave/locales/en-US/app.properties]: There might be better ones like "Scripts Blocked", "Scripts are blocked", "NoScripts enabled", "click here to allow scripts", etc...

File: less/navigationBar.less [hunk 2]
```diff
@@ -284,7 +284,6 @@
 
   .noScript {
     font-size: 16px;
-    cursor: pointer;
```
Comment (luixxiul) [inline less/navigationBar.less]: No other elements in ``.navigatorWrapper`` are styled with this.

File: js/about/bookmarks.js [hunk 3]
```diff
@@ -203,7 +203,7 @@ class AboutBookmarks extends React.Component {
     return <div className='bookmarksPage'>
       <h2 data-l10n-id='folders' />
       <div className='bookmarkPageContent'>
-        <Sticky enabled={true} top={10}>
+        <Sticky enabled top={10}>
```
Comment (luixxiul) [inline js/about/bookmarks.js]: This fixes the lint error.

**Comment by bbondy** [comment]
Thanks

**Pull request merged**